checks.xml

```diff
@@ -86,7 +86,7 @@
             <property name="allowNonPrintableEscapes" value="true"/>
         </module>
         <module name="LineLength">
-            <property name="max" value="100"/>
+            <property name="max" value="120"/>
```

Comment:
I agree that 120 is more preferable to 100 but can we defer this for now and this PR?

Reply:
why? is that can affect anything?

Reply:
its related changes was cluttering this PR. also bring idea to a consensus in the team before making the change as most of our codebase is based on 100. It will lead to great style change across the code base.

Reply:
Ok, let's discuss on Monday.